service/FactomCliFactory.js

```diff
@@ -0,0 +1,77 @@
+const { FactomIdentityManager } = require('factom-identity-lib').app;
+const { FactomCli } = require('factom');
+const { InvalidNetworkParameterError } = require('./errors');
+
+const Networks = Object.freeze({
+    TEST: 'test',
+    MAIN: 'main',
+});
+
+class FactomCliFactory {
```

Comment:
Hmzz, the name is a bit confusing. A Factory constructs multiple related objects, but this class has state.
It is more a registry type of class. I would either rename, or have the actual creation being done in the Factory (witouth state) and then have a registry that is filled using the Factory

Reply:
Ok, I moved it to a registry and made the configs constants. Let me know how it looks now.